src/components/btc-stx-block-links.tsx

```diff
@@ -0,0 +1,102 @@
+import { FC, Fragment } from 'react';
+import { FaBitcoin } from 'react-icons/fa';
+import { BsArrowRightShort } from 'react-icons/bs';
+import { Box, Text, color } from '@stacks/ui';
+import { css } from '@emotion/react';
+import { IconArrowRight } from '@tabler/icons';
+import { buildUrl } from '@components/links';
+import { useRouter } from 'next/router';
+import { StxInline } from '@components/icons/stx-inline';
+import { Circle } from '@components/circle';
+import { useAppSelector } from '@common/state/hooks';
+import { selectActiveNetwork } from '@common/state/network-slice';
+
+interface BtcStxBlockLinksProps {
+  btcBlockHeight?: number;
+  stxBlockHeight: number;
+  stxBlockHash: string;
+  fontSize?: string;
+}
+
+const wrapperStyle = css`
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+`;
+
+const iconStyle = css`
+  margin-right: 3px;
+`;
+
+const linkStyle = (secondary?: boolean, clickable?: boolean, fontSize?: string) => css`
+  font-size: 14px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  font-weight: 500;
+  ${fontSize ? `font-size: ${fontSize};` : 'font-size: 14px;'}
+  ${clickable === true &&
+  `
+    cursor: pointer;
+    &:hover {
+      text-decoration: underline;
+    }
+  `}
+  ${secondary === true &&
+  `
+    color: #747478;
+    font-size: 12px;
+  `}
+`;
+
+const arrowStyle = css`
+  margin: 0 8px;
+`;
+
+export const BtcStxBlockLinks: FC<BtcStxBlockLinksProps> = ({
+  btcBlockHeight,
+  stxBlockHeight,
+  stxBlockHash,
+  fontSize,
+}) => {
+  const router = useRouter();
+  const activeNetworkUrl = useAppSelector(selectActiveNetwork).url;
+
+  return (
+    <Box css={wrapperStyle}>
+      <Circle size="19px" bg={color('accent')} css={iconStyle}>
+        <StxInline strokeWidth={2} size="11px" color="white" />
+      </Circle>
+      <Box
+        css={linkStyle(false, stxBlockHash !== '', fontSize)}
+        onClick={e => {
+          if (stxBlockHash !== '') {
+            e.preventDefault();
+            void router.push(
+              buildUrl(`/block/${encodeURIComponent(stxBlockHash)}`, activeNetworkUrl)
+            );
+          }
+        }}
+      >
+        #{stxBlockHeight}
+      </Box>
+      {btcBlockHeight && (
+        <Fragment>
+          <IconArrowRight size="14px" color={'#747478'} css={arrowStyle} />
+          <FaBitcoin color={'#f7931a'} size={19} css={iconStyle} />
+          <Box
+            css={linkStyle(true, true)}
+            onClick={e => {
+              e.preventDefault();
+              window
```

Comment:
There could be a security issue here as the third party website can have access to the `window.opener.location` and then change it to 'impersonate' our explorer.
The simpler way to fix this would be to use a <a> with a `rel="noopener"`, and it will also be a more semantic html.

```html
<a href="..." target="_blank" rel="noopener">

```

If it's really problematic to change it into a `<a>`, you can do this
```javascript
const win = window.open("XXX");
win.opener = null;
```

Reply:
unfortunately I can't use <a/> here because the parent is also <a/> (nested links are illegal)

Reply:
but you can still set the opener to null to prevent the third party from accessing the window.opener.location

Reply:
I used `window.open(url,'_blank','noopener')`

Reply:
My bad ! I did not see it. Awesome !